drivers/base/bootvid/precomp.h

```diff
@@ -41,16 +49,14 @@ typedef struct tagBITMAPINFOHEADER
 #define BI_RGB  0
 #define BI_RLE4 2
 
-typedef struct _PALETTE_ENTRY
-{
-    UCHAR Red;
-    UCHAR Green;
-    UCHAR Blue;
-} PALETTE_ENTRY, *PPALETTE_ENTRY;
+typedef ULONG RGBQUAD;
 
-VOID
-NTAPI
-InitializePalette(VOID);
+#define RGB(r, g, b)    ((RGBQUAD)(((UCHAR)(b) | ((USHORT)((UCHAR)(g))<<8)) | (((ULONG)(UCHAR)(r))<<16)))
+#define GetRed(quad)    ((UCHAR)(((quad)>>16) & 0xFF))
```

Comment:
OK then since we used these wingdi inspired macros/types, you could name these macros: GetR/G/BValue:
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/psdk/wingdi.h;hb=909f50a857559ce3c69f1cc50d5b864357aec9c9#l2934